tools/abel/README.md

```diff
@@ -1,191 +1,233 @@
-# ABEL
+# ABEL User Guide
 
-ABEL is an acronym that stands for **A**utomated **B**uilding **E**ntity **L**oader. It is a tool that assists systems integrators (and other DBO users) in efficiently creating and
-modifying [Building Configuration YAML files](../../ontology/docs/building_config.md).
+### What is ABEL?
+
+ABEL, or **A**utomated **B**uilding **E**ntity **L**oader, is a tool that assists systems integrators (and other DBO users) in efficiently creating and modifying [Building Configuration YAML files](../../ontology/docs/building_config.md) through use of a [spreadsheet](../../tools/abel/validators/README.md).
+
+Building configuration files contain information about equipment, devices, and spaces within a building such as their device type, input and output points (fields) used to control them, and their relationship to one another. Building configuration files are onboarded using [DB API](https://docs.google.com/document/d/13dzEmBAwz5Dtcx35qs0GXcKWi1QR4Bitw92wW6H54VE/edit?tab=t.0#heading=h.10pbm1zd8jm8) in order to create a digital representation of the devices in the cloud and make use of the data coming from their associated fields. Due to the sheer size of building configuration files (a complete building configuration file for a building can contain thousands of devices with tens of thousands of fields), creating or editing one manually in YAML format can be very time consuming and prone to mistakes. ABEL streamlines this process by allowing the systems integrator to create and edit these files in a spreadsheet format to make it quicker and easier to edit multiple entities or fields at once.
 
 ABEL has the following key features:
-* Convert a Google Sheet (in the ABEL Spreadsheet Template) to a building configuration YAML file
-  * Run Instance Validator (with optional telemetry validation) on the building configuration file
-* Convert a building configuration YAML file to a Google Sheet
 
-## Table of Contents
+* Create and edit a Google Sheet (in the ABEL Spreadsheet Template) and export it to a building configuration file for onboarding.
+ * Run Instance Validator (with optional telemetry validation) on the building configuration file during the conversion process to ensure there are no errors in the file prior to attempting onboarding.
+* Convert a building configuration file to a Google Sheet to make edits on.
+* Split a building configuration file to separate out a single namespace to make it easier to focus edits and onboarding processes to a single area as needed.
```

Comment:
Splitting is going to be removed. Might be worth removing from the documentation.